test/test_suites/suite_vdiff_holtslag_boville.xml

```diff
@@ -0,0 +1,95 @@
+  which were formerly embedded into the vertical diffusion solver.
+-->
+<suite name="vdiff_holtslag_boville" version="1.0">
+  <group name="physics_before_coupler">
+    <scheme>initialize_constituents</scheme>
+    <scheme>to_be_ccppized_temporary</scheme>
+
+    <!-- read namelist options -->
+    <scheme>holtslag_boville_diff_options</scheme>
+    <scheme>vertical_diffusion_options</scheme>
+
+    <!-- stub: zero UBC -->
+    <scheme>zero_upper_boundary_condition</scheme>
+
+    <!-- init phase only: for setting vertical diffusion top -->
+    <scheme>hb_diff_set_vertical_diffusion_top</scheme>
+
+    <!-- hb pre-interstitial: tautotx/y -->
+    <scheme>tms_beljaars_zero_stub</scheme> <!-- stub as TMS/Beljaars not implemented -->
+    <scheme>hb_diff_set_total_surface_stress</scheme>
+
+    <!-- vdiff pre-interstitial 0: create p coords1d, copy surface fluxes from coupler (cam_in) incl. wv fluxes for hb kinematic wv flx. -->
+    <scheme>hb_diff_prepare_vertical_diffusion_inputs</scheme>
+
+    <!-- Initialize Holtslag-Boville boundary layer scheme (no run phase) -->
+    <scheme>holtslag_boville_diff</scheme>
+
+    <!-- Holtslag-Boville boundary layer scheme -->
+    <scheme>hb_pbl_independent_coefficients</scheme>
+    <scheme>hb_pbl_dependent_coefficients</scheme>
+    <scheme>hb_diff_exchange_coefficients</scheme>
+
+    <!-- sponge layer vertical diffusion (mutate kvm) -->
+    <scheme>vertical_diffusion_sponge_layer</scheme>
+
+    <!-- HB scheme diagnostics (after KVM changed by sponge layer scheme) -->
+    <scheme>holtslag_boville_diff_diagnostics</scheme>
+
+    <!-- vdiff pre-interstitial 1: interpolate t, rho, p to interfaces -->
+    <!-- not in WACCM-X mode so do not use rairv in vdiff. -->
+    <scheme>vertical_diffusion_not_use_rairv</scheme>
+    <scheme>vertical_diffusion_set_temperature_at_toa_default</scheme>
+    <scheme>vertical_diffusion_interpolate_to_interfaces</scheme>
+
+    <!-- vdiff pre-interstitial 2: calculate drag coefficient -->
+    <scheme>implicit_surface_stress_add_drag_coefficient</scheme>
+    <!--<scheme>turbulent_mountain_stress_add_drag_coefficient</scheme>-->
+
+    <!-- vdiff pre-interstitial 3: calculate damping rate from drag coef. -->
+    <scheme>vertical_diffusion_wind_damping_rate</scheme>
+    <!--<scheme>beljaars_add_wind_damping_rate</scheme>-->
+
+    <!-- vdiff compute 1: horizontal momentum diffusion; dissipate KE -->
+    <!-- note: vdiff compute schemes calculate "provisional" updates,
+         but these updates are not applied as physics tendencies until the end. -->
+    <scheme>vertical_diffusion_diffuse_horizontal_momentum</scheme>
+
+    <!-- // wip // vdiff mid-interstitial 1? 2? 3?: update surface stresses from horiz. momentum -->
+    <!--<scheme>vertical_diffusion_update_surface_stress</scheme>-->
+    <!--<scheme>turbulent_mountain_stress_add_updated_surface_stress</scheme>-->
+    <!--<scheme>beljaars_add_updated_residual_stress</scheme>-->
```

Comment:
Are you fairly confident that schemes like these will be added to this test suite as part of the gravity wave CCPP-ization?  If not then I might vote to get rid of this code block (just to make sure that it doesn't hang around long after we've forgotten about it).